secret_hitler.py

```diff
@@ -1026,6 +1034,51 @@ def handle_message(self, chat_id, from_player, command, args=""):
                         return "Successfully changed name to '{}'".format(new_name)
             else:
                 return "Must be in game to change nickname"
+        elif command == "claim":
+            if from_player in self.players:
+                if args == "" or args in ["FFF", "FFL", "FLF", "LFF", "FLL", "LFL", "LLF", "LLL", "FF", "FL", "LF", "LL"]:
+                    return "Must specify claim like this: `/claim FFL` (read from left to right as: “I discarded F, my chancellor discarded F, and we enacted an L together.”)"
+                elif len(args) == 3:
+                    # Find the first legislation where from_player was president and didn’t issue a claim
+                    potential_index = -1
+                    for index, (log_line, known_to) in enumerate(self.logs):
+                        if log_line.startsWith("President "+from_player.name+" peeks"):
+                            potential_index = index
+                        elif log_line.startsWith("Chancellor") and index == potential_index + 1:
+                            self.record_log("President {} claims {} ↦ {}".format(player.name, args, args[1:]), known_to=[from_player], position=index)
```

Comment:
`player` is undefined here, did you mean `from_player`?

```suggestion
                            self.record_log("President {} claims {} ↦ {}".format(from_player.name, args, args[1:]), known_to=[from_player], position=index)
```